src/Pages/Main/CommonNav/CommonNav.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import SubMenu from "../SubMenu/SubMenu";
+import "./CommonNav.scss";
+
+class CommonNav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      SubMenuDisplay: "none",
+    };
+  }
+
+  handleSubMenu = (e) => {
+    this.setState({
+      SubMenuDisplay: this.state.SubMenuDisplay === "none" ? "" : "none",
+    });
+  };
+
+  render() {
+    return (
+      <div className="commonNav">
```

Comment:
```suggestion
      <div className="CommonNav">
```
1. 컴포넌트 최상위 태그의 클래스명은 대문자로 써주세요
2. 최상위 태그가 nav가 아니라 div으로 한번 더 감싼 이유가 있을까요?